src/models/converters/transactions/tests/details.rs

```diff
@@ -70,10 +70,12 @@ fn multisig_custom_transaction_to_transaction_details() {
                     MultisigConfirmation {
                         signer: String::from("0x65F8236309e5A99Ff0d129d04E486EBCE20DC7B0"),
                         signature: Some(String::from("0x83b1506c409918f21031e93ed2f62310a5e0c05b1be89242a6a266a7de4af7bc6094e206b33387b8d4465af6087a4d2158815e613aeb186d88d9a1973e00bbe81b")),
+                        submission_time: multisig_tx.confirmations.as_ref().and_then(|confirmation| confirmation.get(0)?.submission_date.map(|it| it.timestamp_millis())),
```

Comment:
I would have hardcoded the number :D

Reply:
I don't know how to do that in tests :/ parsing dates with `chrono` is a bit of a pain. I would like to clean that up eventually.

Reply:
what do you mean? you know the expected number by converting the string `2020-06-22T14:58:34.055882Z` with some tool (e.g. https://codechi.com/dev-tools/date-to-millisecond-calculators/) ... and then you get: `1592837914055`

Reply:
aaah, I was trying to find how the deserialiser turns them into `DateTime<Utc>` , that's much better, thanks!